pkg/resources/data.go

```diff
@@ -913,6 +913,13 @@ func (d *TemplateData) GetKonnectivityServerArgs() ([]string, error) {
 	return tpl.Spec.ComponentsOverride.KonnectivityProxy.Args, nil
 }
 
+func (d *TemplateData) IsSSHKeysDisabled() bool {
+	if d.KubermaticConfiguration() != nil {
+		return d.KubermaticConfiguration().Spec.FeatureGates[features.DisableUserSSHKey]
+	}
```

Comment:
```
if config := d.KubermaticConfiguration(); config != nil {
   return config.Spec.....
}
```

is more readable [to me at least] and would, if KubermaticConfiguration() was expensive, be faster also.